sympy/functions/combinatorial/numbers.py

```diff
@@ -1860,10 +1860,30 @@ def stirling(n, k, d=None, kind=2, signed=False):
 @cacheit
 def _nT(n, k):
     """Return the partitions of ``n`` items into ``k`` parts. This
-    is used by ``nT`` for the case when ``n`` is an integer."""
+    is used by ``nT`` for the case when ``n`` is an integer.
+    The number of partitions of n into k parts is equal to the
+    number of partitions of n-k, if n >= int(q/2) so to increase
```

Comment:
I think the inequality should be "k >= n - k" instead of "n >= int(q/2)". This is an implementation detail that need not be in the docstring but in a comment immediately before calling partition._partition(n - k). So something like this:

    # The number of partitions of ...
    # number of ...
    if  k >= d:

Reply:
@jksuom  done.